### PR TITLE
Add crowdin statistics

### PR DESCRIPTION
**Pull request by Lineflyer**
Add localization badge to development page

### CODE REVIEW

File: send2cgeo.md [hunk 1]
```diff
@@ -70,4 +70,8 @@ Make sure you have Greasemonkey/Tampermonkey **installed and active** in your br
 The installation should start automatically in Greasemonkey/Tampermonkey if you click on the link below:\\
 [Script installation](https://github.com/cgeo/cgeo/raw/master/send2cgeo/send2cgeo.user.js)
 
+## Configure send2cgeo
```
Comment (Bananeweizen) [inline send2cgeo.md]: This shouldn't be in here.

File: development.md [hunk 2]
```diff
@@ -54,6 +54,10 @@ If you install one of our nightly builds to your device, you will not get suppor
 We are using a crowd sourced translation platform to improve the translation and even include new languages.
 If you want to contribute translations you are invited to do so on this platform!
 
+Localization statistics:
+
+[![Crowdin](https://d322cqt584bo4o.cloudfront.net/cgeo/localized.png)](https://crowdin.com/project/cgeo)
```
Comment (Bananeweizen) [inline development.md]: I can't imagine that this host name based reference is stable. But I cannot check myself in the project settings, as I'm not an admin.

Comment (samueltardieu) [inline development.md]: > But I cannot check myself in the project settings, as I'm not an admin.

You now are.

**Comment by Lineflyer** [comment]
Sorry, recent changes on my repo/master have been automatically added here. Closing this, but if you are fine with that, I can directly add it on master?!

**Comment by Bananeweizen** [comment]
yes

**Pull request closed without merging**